packages/integration/package.json

```diff
@@ -16,9 +16,9 @@
   "dependencies": {
     "@vanilla-extract/css": "^1.7.2",
     "esbuild": "^0.11.16",
-    "eval": "0.1.6",
+    "eval": "0.1.8",
```

Comment:
I remember having issues with bumping `eval`, hence why it's pinned. Not sure if those issues are resolved now but would prefer to change it in a separate PR.